compiler/ExtractTypes.ml

```diff
@@ -822,9 +822,13 @@ let extract_type_decl_register_names (ctx : extraction_ctx) (def : type_decl) :
             | None ->
                 VariantId.mapi
                   (fun variant_id (variant : variant) ->
-                    let name =
+                    let variant_name = 
+                      (match variant.item_meta.rename with
+                        | Some(name) -> name
```

Comment:
The formatting is weird (`Some(name)`). Do you reformat the code before pushing?
@Nadrieril the CI should check that the code is well-formatted correct? Because I don't see the nix CI failing for this one.